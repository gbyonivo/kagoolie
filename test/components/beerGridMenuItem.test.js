import React from 'react';
import renderer from 'react-test-renderer';
import store from '../__testData__/store';
import BeerGridMenuItem from '../../src/components/beerGridMenuItem';

describe('BeerGridMenuItem component', () => {
  it('renders correctly', () => {
    const actual = renderer.create(<BeerGridMenuItem beer={store.beersReducer.beers[0]}/>);
    expect(actual).toMatchSnapshot();
  });
});