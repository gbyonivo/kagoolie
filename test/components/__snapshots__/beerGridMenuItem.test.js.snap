// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BeerGridMenuItem component renders correctly 1`] = `
<div
  className="beerGridMenuItem"
>
  <div
    className="image"
  >
    <img
      alt=""
      src="sdsds"
    />
  </div>
  <div
    className="name"
  >
    pluto
  </div>
</div>
`;
